packages/react-native-bpk-component-banner-alert/src/AnimateAndFade.js

```diff
@@ -101,7 +101,8 @@ class AnimateAndFade extends Component<Props> {
     this.measure(captureHeightAndAnimate);
   }
 
-  componentWillReceiveProps(nextProps: Props) {
+  // eslint-disable-next-line camelcase
+  UNSAFE_componentWillReceiveProps(nextProps: Props) {
```

Comment:
Should we be fixing this now rather than waiting for the point it is removed?

Reply:
We should fix deprecations as soon as possible, this will be an error in the next release so is better to update now while we still have time to catch any problems. Plus this causes a very big an annoying warning.

Reply:
Can we/should we be doing this in this PR? Or do we have an action to do that later?